pkg/api/v1alpha1/vspheredatacenterconfig_types.go

```diff
@@ -126,7 +126,7 @@ func (v *VSphereDatacenterConfig) Marshallable() Marshallable {
 }
 
 func (v *VSphereDatacenterConfig) Validate() error {
-	return nil
+	return v.ValidateFields()
```

Comment:
Can we just rename `ValidateFields` to `Validate`?

Reply:
we already have the [Validate](https://github.com/aws/eks-anywhere/pull/3630/files#diff-20cca6355e1892159530ab0612d0d945cea223ae9aac3a320df2245ab70465b4R128)() method here.